endpoints/openrtb2/video_auction.go

```
@@ -91,38 +126,43 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 	}
 	requestJson, err := ioutil.ReadAll(lr)
 	if err != nil {
-		handleError(&labels, w, []error{err}, &vo)
+		handleError(&labels, w, []error{err}, &vo, &debugLog)
 		return
 	}
 
 	resolvedRequest := requestJson
+	if debugLog.EnableDebug {
+		debugLog.Data = fmt.Sprintf("Request:\n%s", string(requestJson))
+		headerBytes, _ := json.Marshal(r.Header)
```

Please do not ignore the error.

---

Ah missed these error checks. Added!